tests_login_with_email_address/test_visibility_of_elements.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from playwright.sync_api import expect
+
+
+def test_visibility(page):
+    # Go to https://login.pracuj.pl/
+    page.goto("https://login.pracuj.pl/")
+    assert page.is_visible("[data-test=\"input-email\"]")
```

Comment:
Using multiple asserts in a single test is not the best way of doing things. The problem with this is simple: if the first assertion fail, rest of them will not be executed, and you will see only part of the truth. You have a few possibilities here, but the best one is to use something what is called `soft assertions`. Out of the box pytest do not allow this, but you can try to use:
- pytest-soft-assetions plugin but you need to remember to add some additional parameter for command line execution and this is not visible in test code
- assertpy library that offers this functionality
- playwright expect also has soft assertion mode ([expect.soft](https://playwright.dev/docs/test-assertions#soft-assertions))
If you are using playwright, best approach would be to use something what is already there, so use expect.soft... instead regular assertion.

Reply:
I'll start making corrections tonight, thx